plugins/ocm/README.md

```diff
@@ -362,38 +356,18 @@ If you are interested in Resource discovery and do not want any of the front-end
 
    - `<ClusterAllocatableResourceCard />`: This is an entity component displaying allocatable resources on a cluster. For example, see [`.status.allocatable`](https://open-cluster-management.io/concepts/managedcluster/#cluster-heartbeats-and-status) of the `ManagedCluster` resource.
 
-### Development
-
-If you have installed the OCM plugin to the example application in the repository, run the `yarn start` command to access the plugin in the root directory and then navigate to `http://localhost:3000/ocm`.
-
-To start a development setup in isolation with a faster setup and hot reloads, complete the following steps:
-
-1. Run the `ocm-backend` plugin in the `plugins/ocm-backend` directory by executing the following command:
-
-   ```console
-   yarn start
-   ```
-
-2. Run the `ocm` frontend plugin in the `plugins/ocm` directory using the following command:
-
-   ```console
-   yarn start
-   ```
-
-The previous steps are meant for local development and you can find the setup inside the `./dev` directories of the individual plugins.
-
 ## For users
 
 ### Using the OCM plugin in Backstage
 
 The OCM plugin integrates your Backstage instance with multi-cluster engines and displays real-time data from OCM.
 
-#### Prerequisites
+#### Usage Prerequisites
```

Comment:
So I decided to change it to `Usage Prerequisites` mainly because there was already a Prerequisites header above. So when I left both as  `Prerequisites`, the `Prerequisites` in the table of contents (that was recently removed in a previous commit) always defaulted to the first one. Would this behaviour occur in the community website's side bar table of contents if we change it back to prerequisites for both?

Reply:
I don't think so. But if this issue is occurring, then maybe we just remove the prereqs and procedure as headings and keep them bold to highlight the section.
WDYT?

Reply:
Ok that sounds good. I'll first make the modifcations and keep them as headers.